packages/react-components/react-theme/stories/Theme/typography/ThemeTypography.stories.tsx

```diff
@@ -1,51 +1,69 @@
 import * as React from 'react';
-import { teamsLightTheme, typographyStyles } from '@fluentui/react-components';
-import type { TypographyStyle } from '@fluentui/react-components';
-
-const theme = teamsLightTheme;
-
-export const TypographyStyles = () => {
-  // var(--tokenName) => tokenName
-  function formatTypographyStyleValue(typographyStyleValue: TypographyStyle) {
-    return (
-      <div>
-        {Object.values(typographyStyleValue).map(value => (
-          <div key={value}>{value.replace(/var\(--(.+)\)/, '$1')}</div>
-        ))}
-      </div>
-    );
-  }
-
-  // caption1Strong => Caption 1 Strong
-  function formatTypographyStyleName(typographyStyleName: string) {
-    return typographyStyleName.replace(/([A-Z\d])/g, ' $1').replace(/^(.)/, firstChar => firstChar.toUpperCase());
-  }
+import { makeStyles, Subtitle2Stronger, Text, typographyStyles } from '@fluentui/react-components';
+import type { TypographyStyles } from '@fluentui/react-components';
+
+type TypographyTokens = [token: keyof TypographyStyles, tokenName: string, values: string[]][];
+
+const useStyles = makeStyles({
+  container: {
+    rowGap: '24px',
+    columnGap: '48px',
```

Comment:
Is it ok that we use pixels here?

Reply:
The unit that we use for the library is px, especially for the documentation. :)